back/engines/free/email_campaigns/spec/acceptance/consents_spec.rb

```diff
@@ -58,21 +58,15 @@
         json_response = json_parse(response_body)
         multiloc_service ||= MultilocService.new
 
-        campaigns_mgmt_attrs = @campaigns.map do |campaign|
-          [
-            [:recipient_role_multiloc, multiloc_service.i18n_to_multiloc(campaign.class.recipient_role_multiloc_key).transform_keys(&:to_sym)],
```

Comment:
Why was this removed? Is it not needed?

Reply:
i removed the recipient role grouping for consents. its going to be just content type